src/components/AssetInfo/TokenSideBarContent.tsx

```diff
@@ -20,9 +21,18 @@ const TokenSideBarContent = ({
   const isEqualBeneficiaryAndHolder = adminAddress === holderAddress && adminAddress === beneficiaryAddress;
   const showHolder = adminAddress === holderAddress || isEqualBeneficiaryAndHolder;
   const showBeneficiary = adminAddress === beneficiaryAddress && !isEqualBeneficiaryAndHolder;
+  const showLoader = holderAddress === "" && beneficiaryAddress === "";
+  const showNoAccess = showLoader === false && (adminAddress !== holderAddress && adminAddress !== beneficiaryAddress);
```

Comment:
refactor this code based on the api reponse. rather than just checking the empty. show the error if api response is empty.